Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -106,7 +106,6 @@ extension ClangModule {
         case .directory(let path):
             output += "    umbrella \"\(path)\"\n"
         }
-        output += "    link \"\(c99name)\"\n"
```

Comment:
If we are generating framework targets in Xcode, it is probably important to declare the module as "framework module <name>".